--- t/13-errors.t~	Sun Oct  6 21:10:53 2002
+++ t/13-errors.t	Sun Oct 13 17:04:00 2002
@@ -66,25 +66,31 @@
 
 #------------------------------------------------------------
 
-    $group->add_support( path => '/support/unreadable',
-			 component => <<'EOF',
+    # This fails as root because the file will always be readable, but
+    # we can't know that it will fail until we're inside the test.  So
+    # we'll just run this test for developers, not end users.
+    if ( $ENV{MASON_MAINTAINER} )
+    {
+        $group->add_support( path => '/support/unreadable',
+                             component => <<'EOF',
 unreadable
 EOF
-		       );
+                           );
 
-    my $file = File::Spec->catfile( $group->comp_root, 'errors', 'support', 'unreadable' );
+        my $file = File::Spec->catfile( $group->comp_root, 'errors', 'support', 'unreadable' );
 
-    $group->add_test( name => 'cannot_read_source',
-		      description => 'Make sure that Mason throws a useful error when it cannot read a source file',
-		      component => <<"EOF",
+        $group->add_test( name => 'cannot_read_source',
+                          description => 'Make sure that Mason throws a useful error when it cannot read a source file',
+                          component => <<"EOF",
 <%init>
 chmod 0222, '$file'
-    or die "Cannot chmod $file to 0222: \$!";
+    or die "Cannot chmod \\Q$file\\E: \$!";
 \$m->comp('support/unreadable');
 </%init>
 EOF
-		      expect_error => q|Permission denied|
-		    );
+                          expect_error => q|Permission denied|
+                        );
+    }
 
 #------------------------------------------------------------
 
