--- HTML-Mason-1.015/Makefile.PL~	Wed Apr 18 00:53:18 2001
+++ HTML-Mason-1.015/Makefile.PL	Wed Apr 18 00:56:19 2001
@@ -21,10 +21,10 @@
 	. "    from CPAN/modules/by-authors/id/D/DR/DROLSKY/Params-Validate-x.x.tar.gz\n\n";
 
 make_config();
-setup_mod_perl_tests();
+#setup_mod_perl_tests();
 
 # Get these into the MY namespace for the MY::test method
-%MY::APACHE = %APACHE;
+#%MY::APACHE = %APACHE;
 
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
